pkg/drivers/kic/oci/oci.go

```diff
@@ -51,9 +53,18 @@ func DeleteAllContainersByLabel(ociBin string, label string) []error {
 		return nil
 	}
 	for _, c := range cs {
-		cmd := exec.Command(ociBin, "rm", "-f", "-v", c)
-		if out, err := cmd.CombinedOutput(); err != nil {
-			deleteErrs = append(deleteErrs, errors.Wrapf(err, "delete container %s: output %s", c, out))
+		_, err := containerStatus(ociBin, c)
+		// only try to delete if docker/podman inspect returns
+		// if it doesn't it means docker-daemon is stuck and needs restart
+		if err != nil {
+			deleteErrs = append(deleteErrs, errors.Wrapf(err, "delete container %s: -daemon is stuck.", c, ociBin))
+			glog.Errorf("%s-daemon seems to be stuck. Please try restarting your %s.", ociBin, ociBin)
```

Comment:
`docker-daemon` isn't a thing. Perhaps this should say something like:

`Your %s daemon appears to be stuck. Please try restarting %s.`